demo/sdl_opengl3/main.c

```diff
@@ -140,6 +146,11 @@ int main(int argc, char *argv[])
     #endif
     #endif
 
+    #ifdef INCLUDE_CONFIGURATOR
+    static struct nk_color color_table[NK_COLOR_COUNT];
```

Comment:
This should be at the beginning of the `main()` with other declarations. It does not pass the -pedantic flag on Linux

Reply:
Done.  I had been testing locally with a modified Makefile to make my life easier.